packages/devtools_app/test/integration_tests/integration.dart

```diff
@@ -385,28 +395,23 @@ class WebdevFixture {
       environment['DART_VM_OPTIONS'] = '';
     }
 
-    // Run the snapshot directly instead of going via pub.bat so that on Windows
-    // when we send kill() it gets passed to Dart and doesn't sometimes terminate
-    // the shell and leave the Dart process behind.
-    final executable = Platform.executable;
-    final pubSnapshotPath = path.join(
-      File(executable).parent.path,
-      'snapshots',
-      'pub.dart.snapshot',
-    );
-
-    final List<String> cliArgs = [pubSnapshotPath, 'global', 'run', 'webdev']
-        .followedBy(buildArgs)
-        .toList();
+    // TODO(djshuckerow): The pub-based version of this code would run a pub
```

Comment:
nit: file a bug to track this. @DanTup may have some more context and ideas on fixing this.

Reply:
I don't know of any much better options here :( In Dart-Code we tend to try and get the `pid` out of processes (most of the tools will give their pid somewhere in one of the early requests) and then kill that instead. We do also run Pub via its snapshot, that's mostly to allow us to use `--pause-isolates-on-start` because the pub mechanism for the same behaves differently and we want to use the same code for Dart/Pub debugging.

Reply:
#1145